activation/atxdb.go

```diff
@@ -710,6 +731,11 @@ func (db *DB) GetFullAtx(id types.ATXID) (*types.ActivationTx, error) {
 	return atx, nil
 }
 
+// GetAtxIterByCoinbase returns an iterator over all ATXs associated with that coinbase address
+func (db *DB) GetAtxIterByCoinbase(coinbase types.Address) database.Iterator {
+	return db.atxs.Find(getAtxCoinbasePrefix(coinbase))
```

Comment:
Out of curiosity what happens here if the coinbase that's passed in is empty, or invalid, or there's just no matching data? It would be good to add tests for each of these cases